Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.6-stretch
+
+RUN pip3 install git+https://github.com/Cobliteam/kafka-topic-dumper.git
+
+WORKDIR /opt/kafka-topic-dumper
+
+VOLUME /data
+
+ENTRYPOINT ["kafka-topic-dumper"]
```

Comment:
We should not set the entry point to the dumper script, otherwise we won't be able to call any other executables. Just setting the CMD is enough.